notebooks/official/generative_ai/text_embedding_new_api.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Could you please add more comments to explain what this method does?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>For example, what each line of code is trying to accomplish. What are the inputs and outputs? What does it mean? How they can retrieve and use the output?</p>

Reply:
Added more comments.